mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb

```diff
@@ -381,32 +378,59 @@ func flattenEnableDisplay(displayDevice *computeBeta.DisplayDevice) interface{}
 
 // Terraform doesn't correctly calculate changes on schema.Set, so we do it manually
 // https://github.com/hashicorp/terraform-plugin-sdk/issues/98
-func schedulingHasChange(d *schema.ResourceData) bool {
+// 1st bool is a change that requires a reboot and the second bool is a change that doesn't
```

Comment:
Is this a common pattern already? To me returning two booleans is pretty confusing, and could easily be misused in the future. Wrapping these in an object could make it more clear what each one of them means

Reply:
Or this could be two individual methods, `HasChangeRequiringReboot` and not

Reply:
You're right, I like splitting it into two methods